alignak/shinken_import_hook.py

```diff
@@ -1,5 +1,5 @@
-
 # -*- coding: utf-8 -*-
```

Comment:
it's ok : https://www.python.org/dev/peps/pep-0263/ : 

```To define a source code encoding, a magic comment must
    be placed into the source files either as first or second
    line in the file```

second line is clearly for the case where you use a shebang .. so in this case (no shebang needed) the good/better choice is the first one ;)